internal/install/execution/install_status.go

```diff
@@ -61,126 +61,134 @@ func NewStatusRollup(reporters []StatusReporter) *StatusRollup {
 	return &s
 }
 
-func (s *StatusRollup) ReportRecipeAvailable(recipe types.Recipe) {
+func (s *InstallStatus) RecipeAvailable(recipe types.Recipe) {
 	s.withAvailableRecipe(recipe)
 
 	for _, r := range s.statusReporters {
-		if err := r.ReportRecipeAvailable(s, recipe); err != nil {
+		if err := r.RecipeAvailable(s, recipe); err != nil {
 			log.Errorf("Could not report recipe execution status: %s", err)
 		}
 	}
 }
 
-func (s *StatusRollup) ReportRecipesAvailable(recipes []types.Recipe) {
+func (s *InstallStatus) RecipesAvailable(recipes []types.Recipe) {
 	s.withAvailableRecipes(recipes)
 
 	for _, r := range s.statusReporters {
-		if err := r.ReportRecipesAvailable(s, recipes); err != nil {
+		if err := r.RecipesAvailable(s, recipes); err != nil {
 			log.Errorf("Could not report recipe execution status: %s", err)
 		}
 	}
 }
 
-func (s *StatusRollup) ReportRecipeInstalled(event RecipeStatusEvent) {
-	s.withRecipeEvent(event, StatusTypes.INSTALLED)
+func (s *InstallStatus) RecipesSelected(recipes []types.Recipe) {
```

Comment:
Is the method name a question here, or an action being taken?

Reply:
This looks like the diff tool struggling.  `RecipesSelected` has been added, and `ReportRecipeInstalled` has also been renamed to `RecipeInstalled`.